gpiozero/boards.py

```diff
@@ -1156,6 +1156,8 @@ def close(self):
         if self._master:
             m = self._master
             self._master = None
+            self._socket = None
+            self._value = None
```

Comment:
Without these, after you close a socket, methods socket and value still return a valid number.. Didn't seem right to me.